core/controllers/editor_test.py

```diff
@@ -525,7 +527,7 @@ def test_exploration_download_handler_for_default_exploration(self):
         response = self.get_custom_response(download_url, 'text/plain')
 
         # Check downloaded zip file.
-        filename = 'oppia-ThetitleforZIPdownloadhandlertest!-v2.zip'
+        filename = 'b\'oppia-ThetitleforZIPdownloadhandlertest!-v2.zip\''
```

Comment:
This looks like you have 'attachment; filename=b\'...\''. That seems odd -- the "b" should not appear in the response.headers string at all.

Reply:
The b gets added as we replaced unicode with builtins.str (or python_utils.str). This cannot be removed. Any solution what to do?

Reply:
I don't understand why it cannot be removed. It seems to me that whether b gets added or not is an internal detail that is within our control. We should be able to change how we handle processing of the header string so that the correct thing is outputted.

Reply:
One option that remained was to use decode on the whole title string. I did that

Reply:
It looks reasonable to me. Can you test that it works fine in the history tab (i.e. when you download the file, the filename looks sensible?)

Once you've done that, I think that's it for this PR, assuming that the automated checks pass :)

Reply:
Yea tested it with both str and unicode characters. Works good with both of them

Reply:
Just to check -- could you please explain what specific steps you took to test this?

Reply:
I created an exploration first with title 'xxx' and then downloaded the exploration by going to /createhandler/download/exp_id. The filename was correctly shown - - oppia-xxx-v3.zip(as far as I remember).

Next I tried creating an exploration with title ¡Hola! and then again downloaded the exploration. The filename this time was oppia-Hola!-v3.zip. The unicode letter got escaped. I guess this is what we wanted?

Reply:
Yes, that sounds good. Thanks! LGTM and merging.